ros2_batch_job/packaging.py

```diff
@@ -73,10 +73,10 @@ def build_and_package(args, job):
             '--only-packages', 'ros1_bridge',
         ] + (['--isolated'] if args.isolated else []) +
             (
-                ['--cmake-args', '-DCMAKE_BUILD_TYPE=' + args.cmake_build_type]
+                ['--cmake-args', '-DCMAKE_BUILD_TYPE=' + args.cmake_build_type + ' -- ']
                 if args.cmake_build_type else []
             ) + [
-            '--make-flags', '-j1'
+            '--make-flags', '-j1', ' -- '
```

Comment:
Yes, all these strings are items in a list which is later joined with spaces. So none of the items need spaces around them.